idc.Makefile

```diff
@@ -23,3 +29,14 @@ destroy-state:
 composer-install:
 	echo "Installing via composer"
 	docker-compose exec drupal with-contenv bash -lc 'COMPOSER_MEMORY_LIMIT=-1 composer install'
+
+.PHONY: snapshot-image
+.SILENT: snapshot-image
+snapshot-image:
+	docker-compose stop
+	docker run --rm --volumes-from snapshot \
```

Comment:
After a `mkdir snapshot/data` and `docker-compose build snapshot`, this fails:
```
$ make -B snapshot-image
Unable to find image 'alpine:latest' locally
latest: Pulling from library/alpine
Digest: sha256:185518070891758909c9f839cf4ca393ee977ac378609f700f60a771a2dfe321
Status: Image is up to date for alpine:latest
docker: Error response from daemon: No such container: snapshot.
See 'docker run --help'.
make: *** [snapshot-image] Error 125
```

I think the main reason is that no _container_ named `snapshot` exists.  `docker run` is ignorant of `docker-compose.yml`, but `docker-compose run` doesn't support `--volumes-from`.

In order to create one, I had to manually run:
`docker-compose run --name snapshot snapshot`

And then re-invoke `make -B snapshot-image`

Reply:
Fascinating.  There is a docker compose directive:
```
  snapshot:
    build:
      context: /home/birkland/projects/idc-isle-dc/snapshot
    container_name: snapshot
```

Does it ignore that and name the snapshot container something else?

Reply:
Your command is invoking `docker`, not `docker-compose`, so nothing in `docker-compose.yml` is used.

Reply:
That's correct, it's invoking docker to launch a container that dumps a tar file.  However, as an argument (`--volumes-from`), it is referencing the name of a container that is presumed to have been created by docker-compose.  docker-compose has been instructed (via `container_name`) to create a container with that name.  So the question is:  why isn't a container with that name available to you?

my theory is that it's one of the following:
* the stack wasn't running
* docker-compose is creating the `snapshot` container under a different name than is specified via `container_name`.

I haven't seen this, so I'm curious what is happening for you